src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/mms/MMSDeltaProjectEventListenerAdapter.java

```diff
@@ -245,34 +245,44 @@ private void determineChangesUsingCommitResponse(File responseFile, Set<String>
             Set<ObjectNode> commitObjects = parsedResponseObjects.get(MDKConstants.COMMITS_NODE);
 
             if(commitObjects != null && !commitObjects.isEmpty()) {
-                int size = 0;
+                Map<String, Integer> commitSizes = new HashMap<>();
                 String commitSyncDirection = "";
                 for(ObjectNode jsonObject : commitObjects) {
                     if(jsonObject.isArray() && jsonObject.size() > 0) {
                         for(int i = 0; i < jsonObject.size(); i++) {
                             JsonNode currentNode = jsonObject.get(i);
-                            JsonNode sourceField = currentNode.get(MDKConstants.SOURCE_FIELD);
-                            boolean isSyncingCommit = sourceField != null && sourceField.isTextual() && MDKConstants.MAGICDRAW_SOURCE_VALUE.equalsIgnoreCase(sourceField.asText());
-                            if(isSyncingCommit) {
-                                commitSyncDirection = "Removed";
-                            } else {
-                                commitSyncDirection = "Added";
-                            }
-
-                            size = validateJsonElementArray(currentNode, isSyncingCommit, lockedElementIds, size);
+                            validateJsonElementArray(currentNode, lockedElementIds, commitSizes);
                         }
                     } else { // what do we throw here? and should we?
-                        throw new IllegalStateException();
+                        validateJsonElementArray(jsonObject, lockedElementIds, commitSizes);
```

Comment:
Given https://github.com/Open-MBEE/mdk/pull/222#discussion_r807262584, we can revert this. We only want to support a single response shape.